app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true }
+  validates :likes_counter, numericality: { only_integer: true }
```

Comment:
- [ ] Same for the `comments_counter` and `likes_counter`, kindly make sure the value of these columns are integers greater than or equal to zero only